### PR TITLE
Update debian/changelog for the 1.0 release.

### PR DESCRIPTION
**Pull request by kyrofa**
Only merge if this is the last commit for a 1.0 release.

### CODE REVIEW

**Comment by sergiusens** [comment]
Nice work getting started, this is missing the (LP: #xxxxx) mentions though (check the previous changelog entries for reference).

**Comment by kyrofa** [comment]
Waiting for #204 and #206 to be backported, then this will be ready.

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+  [ Sergio Schvezov ]
+  * Wrapping exe should not be over match to replace
+
+  [ Alexander Sack ]
+  * use archive.apache.org URL to ensure we refer to a stable URL that does not disappear on latest release
+
+  [ Daniel Holbach ]
+  * add doc about the syntax of snapcraft.yaml
+
+  [ Kyle Fazzari ]
```
Comment (sergiusens) [inline debian/changelog]: Maybe do

```
* Catking plugin:
  - Add support for ROS tools.
  - Use rosdep for dependency resolution.
  - ...
```

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,27 @@
+snapcraft (1.0) xenial; urgency=medium
+
+  [ Sergio Schvezov ]
+  * Wrapping exe should not be over match to replace
+
+  [ Alexander Sack ]
+  * use archive.apache.org URL to ensure we refer to a stable URL that does not disappear on latest release
+
+  [ Daniel Holbach ]
+  * add doc about the syntax of snapcraft.yaml
```
Comment (sergiusens) [inline debian/changelog]: s/add/Add/

File: debian/changelog [hunk 3]
```diff
@@ -1,3 +1,27 @@
+snapcraft (1.0) xenial; urgency=medium
+
+  [ Sergio Schvezov ]
+  * Wrapping exe should not be over match to replace
+
+  [ Alexander Sack ]
+  * use archive.apache.org URL to ensure we refer to a stable URL that does not disappear on latest release
```
Comment (sergiusens) [inline debian/changelog]: s/use/Use/

**Comment by kyrofa** [comment]
Alright, this is ready from my perspective.

File: debian/changelog [hunk 4]
```diff
@@ -1,3 +1,34 @@
...
+    - Use rosdep for dependency resolution.
+    - Refactor build.
+    - Improve cmake path rewrite.
+  * Python2 plugin:
+    - Make site-packages link relative. (LP: #1523384)
+  * Python3 plugin:
+    - Make site-packages link relative. (LP: #1523384)
+
+  [ Renat Galimov ]
+  * Close not called on the qml plugin
```
Comment (sergiusens) [inline debian/changelog]: Can you create a bug for this?

File: debian/changelog [hunk 5]
```diff
@@ -1,3 +1,34 @@
+snapcraft (1.0) xenial; urgency=medium
+
+  [ Sergio Schvezov ]
+  * Wrapping exe should not be over match to replace (LP: #1523912)
+
+  [ Alexander Sack ]
+  * Use archive.apache.org URL to ensure we refer to a stable URL that does not disappear on latest release
+
+  [ Daniel Holbach ]
+  * Add doc about the syntax of snapcraft.yaml
```
Comment (sergiusens) [inline debian/changelog]: I think there is an open bug for this, maybe link to it?

Comment (kyrofa) [inline debian/changelog]: I looked, I'm coming up empty. Should I make one?

File: debian/changelog (hunk 4 above)
Comment (kyrofa) [inline debian/changelog]: Done.

**Pull request merged**